.github/workflows/build.yaml

```diff
@@ -66,3 +66,23 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  wheel-build:
+    needs: [cpp-build]
```

Comment:
Does the `wheel-build` need to depend on the `cpp-build` (conda)?

Reply:
Is this expected to work or would there be binary incompatibilities? Currently I am just rebuilding the C++ lib in the `build_wheel.sh` script via the `run build_local all` call.

Reply:
Just to clarify, I don't think the `wheel-build` stage needs to depend on the `cpp-build` stage since they are independent, so this line can be removed.

Reply:
sorry, I had misread and thought this was a suggestion to add it. I have removed it now